content/docs/integrations/aws-s3/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Integrating with AWS S3
+publishedDate: '2022-03-29T21:00:00.0Z'
+description: How to configure Roadie to read from S3
+---
+
+## Introduction
+
+Backstage supports reading data from AWS S3 through the AwsS3UrlReader. This guide describes how to set up access for 
+Roadie to your S3 buckets in your infrastructure and configure the AwsS3UrlReader settings.
+
+##  Step 1: Get the roadie IAM details
+
+Navigate to `Administration > Settings > AWS S3` and make a note of the Roadie backend role ARN and account ID.
+
+![Role Details](./role-details.png)
+
+##  Step 2: Create a federated role in your account for Roadie
+
+Follow the steps [here](/docs/details/accessing-aws-resources) to create the role. 
+
+You'll need to attach a policy which allows access to the required S3 buckets such as `AmazonS3ReadOnlyAccess`. This policy grants roadie read access to all buckets. 
+If you do not want to grant this access you can [create your own policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html) 
+which restricts access to only certain buckets. e.g. 
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:Get*",
+        "s3:List*",
+        "s3-object-lambda:Get*",
```

Comment:
I'm not sure these lambda policies are needed. I thought they were for lambdas to be able to listen to S3 bucket object modifications. 

I think 
two statements are needed:
``` 
{
    "Effect": "Allow",
    "Action": [
      "s3:PutObject",
    ]
    "Resource": "THEBUCKET/*"
}
{
   "Effect": "Allow",
   "Action": [
      "s3:ListBucket"
    ]
    "Resource": "THEBUCKET"
  }
```